src/main/resources/templates/aeolus/java/plain_maven.yaml

```diff
@@ -0,0 +1,25 @@
+api: v0.0.1
+metadata:
+    name: java-maven
+    id: java-windfile
+    description: Build Plan for a java Exercise with maven
+    author: Andreas Resch
+    docker:
+        image: ls1tum/artemis-maven-template
+        tag: java17-20
```

Comment:
The image and tag should also be loaded from the application.yml, admins should have the possibility to change that.

Reply:
I removed it